Source/ACE/Entity/Landblock.cs

```diff
@@ -303,6 +303,35 @@ public List<WorldObject> GetWorldObjectsByGuid(ObjectGuid objectguid, bool neigh
             }
             return allworldobj;
         }
+        /// <summary>
+        /// This is going to find a single object by guid and search both the current landblock and all adjacent landblocks.
+        /// If more than one item is found - this should not happen, it is going to return a null and punt. Og II
+        /// </summary>
+        /// <param name="objectGuid"></param>
+        /// <returns></returns>
+        private WorldObject GetWorldObjectByGuid(ObjectGuid objectGuid)
+        {
+            var objectList = GetWorldObjectsByGuid(objectGuid, true);
```

Comment:
See, this is the method you needed, but with the adjacency flag.  Combine this with the bad method above.